tools/gmon/gmon/clients/utils.py

```diff
@@ -0,0 +1,47 @@
+"""
+`utils.py`
+Util functions.
+"""
+
+from google.protobuf.json_format import MessageToDict
+
+
+def decorate(decorator, methods=[]):
+    def inner(cls):
+        for attr in cls.__dict__:  # there's propably a better way to do this
+            attribute = getattr(cls, attr)
+            if callable(attribute) and attribute.__name__ in methods:
+                setattr(cls, attr, decorator(attribute))
+        return cls
+
+    return inner
+
+
+def to_json(func):
+    def inner(*args, **kwargs):
```

Comment:
use [functools.wraps](https://docs.python.org/3/library/functools.html#functools.wraps) so that stacktraces / docs are still readable with original function name.